### ISSUE TITLE
Accessibility module increases the chart size

### ISSUE THREAD

**Issue by kamilkulig** [highcharts/highcharts#10637]
#### Expected behaviour
300px high chart should fit into 300px high iframe: http://jsfiddle.net/BlackLabel/o1um6f7q/

#### Actual behaviour
It doesn't - vertical scrollbar appears.

#### Live demo with steps to reproduce
http://jsfiddle.net/BlackLabel/hezmdck9/

#### Product version
Highcharts 7.1.1

**Comment by TorsteinHonsi** [comment]
@kamilkulig I get the same results when I remove the accessibility module...

http://jsfiddle.net/highcharts/k6q81y0p/

**Comment by pawelfus** [comment]
It's the module in the iframe: http://jsfiddle.net/BlackLabel/58gc2bLk/

**Comment by TorsteinHonsi** [comment]
@oysteinmoseng When removing this element the scrollbar disappears. Since it's `position:absolute`, maybe we can just apply a `top` property to it?

`<h6 tabindex="0" aria-label="End of interactive chart" aria-hidden="false" style="position: absolute; width: 1px; height: 1px; z-index: 0; overflow: hidden; outline: none;"></h6>`

**Comment by kamilkulig** [comment]
But accessibility is still included in fiddle.jshell.net/BlackLabel/nv79xdbh/show script (the one you load into an iframe).

**Comment by oysteinmoseng** [comment]
Since focus is set to this element during keyboard navigation, a `top:0` or similar could cause the page to scroll. We should probably just position this inside the chart container
